android-components/components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/activity/GeckoViewActivityContextDelegate.kt

```diff
@@ -35,7 +35,7 @@ class GeckoViewActivityContextDelegate(
      */
     override fun getActivityContext(): Context? {
         val context = contextRef.get()
-        if ((context == null) || (context as Activity).isDestroyed) {
+        if ((context == null)) {
```

Comment:
Had to remove this casting and check because the context can no longer be an original activity one. Maybe I'm doing something wrong here, but this casting fails even though we're able to use the context (which means its an activity context).